test/sanity/code-smell/type-boilerplate.py

```diff
@@ -0,0 +1,322 @@
+#!/usr/bin/env python
+
+import sys
+
+
+def main():
+    # We want to touch bundled stuff as little as possible
+    skip = frozenset((
+        'lib/ansible/compat/selectors/_selectors2.py',
+        'lib/ansible/module_utils/six/_six.py',
+        'setup.py',
+    ))
+
+    # These files pre-date the boilerplate test and should be fixed up as time goes on
+    prune = (
+        'contrib/inventory/',
+        'contrib/vault/',
+        'docs/',
+        'examples/',
+        'hacking/',
+        'lib/ansible/module_utils/csharp/',
+        'lib/ansible/module_utils/powershell/',
+        'lib/ansible/module_utils/aws/__init__.py',
```

Comment:
Move filenames out of `prune` into `skip` -- just separate them from the other entries with a comment.

Reply:
I can do that as long as we don't care that we then have to read and check for the boilerplate on every file in the skip_list.  (grandfathered currently has to be searched in order to tell people when to remove a file from the grandfathered list).

Reply:
As long as the files in the `prune` list are going to be on a separate `grandfathered` list that will be fine. It just didn't seem right to have files in the `prune` list when it was intended only for directories.